projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -805,20 +679,16 @@ class="<?php echo esc_attr( $classes['email_field'] ); ?> required"
 							<input type="hidden" name="sub-type" value="<?php echo esc_attr( $data['source'] ); ?>"/>
 							<input type="hidden" name="redirect_fragment" value="<?php echo esc_attr( $form_id ); ?>"/>
 							<?php
+							wp_nonce_field( 'blogsub_subscribe_' . $blog_id, '_wpnonce', false );
```

Comment:
That nonce was not present on the Jetpack form. Probably not an issue.

Reply:
Also, it used to be surrounded with:
```
<?php
							if ( is_user_logged_in() ) {
								wp_nonce_field( 'blogsub_subscribe_' . get_current_blog_id(), '_wpnonce', false );
							}
							?>
```
```suggestion
							if ( ( new Host() )->is_wpcom() && is_user_logged_in() ) {
							       wp_nonce_field( 'blogsub_subscribe_' . $blog_id, '_wpnonce', false );
							}
```

Reply:
pushed [this commit](https://github.com/Automattic/jetpack/pull/33865/commits/31eba3884a6ffc1c893e7664afa5f2228e682e6d) so we always verify the nonce too